packages/processes/test/e2e/processes.e2e.test.ts

```diff
@@ -0,0 +1,96 @@
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const result = await service.getConformance();
+  expect(Array.isArray(result.conformsTo)).toBe(true);
+
+  result.conformsTo.forEach((item: string) => {
+    expect(typeof item).toBe('string');
+  });
+});
+
+test('e2e: getProcesses() should return a processes list', async () => {
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const result = await service.getProcesses();
+  expect(Array.isArray(result.processes)).toBe(true);
+
+  result.processes.forEach((item: IProcessSummary) => {
+    expect(item.id).toBeTruthy();
+    expect(item.version).toBeTruthy();
+    expect(item.title).toBeTruthy();
+  });
+});
+
+test('e2e: getProcess() should return a process', async () => {
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const process = await service.getProcess('hello-world');
+
+  expect(process.id).toBeTruthy();
+  expect(process.version).toBeTruthy();
+  expect(process.title).toBeTruthy();
+});
+
+test('e2e: executeJob({ mode: sync }) should execute job synchronously', async () => {
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const result = await service.executeProcess('hello-world', {
+    mode: 'sync',
+    inputs: {
+      name: '@ogcapi-js',
+    },
+    outputs: {
+      echo: {
+        transmissionMode: 'value',
+      },
+    },
+    response: 'document',
+  });
+
+  expect(result).toBeTruthy();
+  // TODO add more assertions when https://github.com/geopython/pygeoapi/discussions/1383 is resolved
+});
+
+// TODO: check again when https://github.com/geopython/pygeoapi/discussions/1383#discussioncomment-7421781 is resolved
+/* test('e2e: executeJob({ mode: async }) should execute job asynchronously', async () => {
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const result = await service.executeProcess('hello-world', {
+    mode: 'async',
+    inputs: {
+      name: '@ogcapi-js'
+    },
+    outputs: {
+      echo: {
+        transmissionMode: 'value',
+      },
+    },
+    response: 'document',
+  });
+
+  console.info({ result });
+
+  expect(result).toBeTruthy();
+}); */
+
+test('e2e: getJobs() should return a jobs list', async () => {
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const result = await service.getJobs();
+
+  expect(Array.isArray(result.jobs)).toBe(true);
+  result.jobs.forEach(job => {
+    expect(job.processID).toBe('hello-world');
+    expect(job.status).toBe('successful');
+  });
+});
+
+test('e2e: getJob() should return a job', async () => {
+  const service = new ProcessesService({ baseUrl: TEST_SITE });
+  const result = await service.getJob('f78fa304-775f-11ee-88b4-0242ac110002');
+
+  expect(result).toBeDefined();
+  expect(result.processID).toBe('hello-world');
+  expect(result.status).toBe('successful');
+  expect(result.jobID).toBe('f78fa304-775f-11ee-88b4-0242ac110002');
+});
```

Comment:
Does these two e2e tests depend on the previous `executeJob` e2e test?

Reply:
No, they do not depend on the previous `executeJob()` tests, because the e2e docker image contains a pre-populated jobs database.

But the assertions inside the `getJobs()` for loop should be less specific. I will improve this test.

Reply:
Test are improved.